packages/babel-generator/test/fixtures/parentheses/break-statement/expected.js

```diff
@@ -0,0 +1,5 @@
+label: for (const a of [1, 2, 3]) {
+  break
+  /*test*/
```

Comment:
Just good formatting. I'd like to keep it if we're in between statements, but it shouldn't apply if we're in an expression.